File: sandbox/ruby/src/SignatureRequestListDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::SignatureRequestApi.new.signature_request_list(
        {
            account_id: nil,
            page: 1,
            page_size: 20,
            query: nil,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling SignatureRequest#signature_request_list: #{e}"
end

File: sandbox/php/src/SignatureRequestListDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

try {
    $response = (new Dropbox\Sign\Api\SignatureRequestApi(config: $config))->signatureRequestList(
        account_id: null,
        page: 1,
        page_size: 20,
        query: null,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling SignatureRequest#signatureRequestList: {$e->getMessage()}";
}
